docs/running-on-kubernetes.md

```diff
@@ -663,11 +663,21 @@ specific to Spark on Kubernetes.
   </td>
 </tr>
 <tr>
-  <td><code>spark.kubernetes.node.selector.[labelKey]</code></td>
+  <td><code>spark.kubernetes.driver.selector.[labelKey]</code></td>
```

Comment:
We should keep `spark.kubernetes.driver.selector.[labelKey]` for backward compatibility. `spark.kubernetes.driver.nodeSelector.[labelKey]` should default to `spark.kubernetes.driver.selector.[labelKey]` if not specified.

Reply:
agreed we should keep it, but recommend annotating it as deprecated